client/src/App.js

```diff
@@ -27,69 +28,71 @@ export default function App() {
     },
     reminders: [
       {
-        reminderId: 1,
+        reminderId: 11111,
         trigger: "Supermarket",
+        triggerEvent: "shopping",
         creationTime: 12343434,
         tasks: [
           {
             taskId: 1,
-            happening: "shopping",
             verb: "buy",
             action: "some milk",
             with: "Max",
           },
           {
             taskId: 2,
-            happening: "shopping",
             verb: "buy",
             action: "some toiletpaper",
             with: "",
           },
           {
             taskId: 3,
-            happening: "shooing",
+            verb: "Ask for",
+            action: "vegan icecream",
+            with: "",
+          },
+          {
+            taskId: 4,
             verb: "Ask for",
             action: "vegan icecream",
             with: "",
           },
         ],
       },
       {
-        reminderId: 2,
+        reminderId: 22222,
         trigger: "Eric",
+        triggerEvent: "is coming over",
         creationTime: 12343434,
         tasks: [
           {
             taskId: 1,
-            happening: "is Visiting",
             verb: "book",
             action: "restaurant table",
             with: "Anne",
           },
         ],
       },
       {
-        reminderId: 1,
+        reminderId: 33333,
         trigger: "Joker",
+        triggerEvent: "attacks Gotham",
         creationTime: 12343434,
         tasks: [
           {
             taskId: 1,
-            happening: "is attacking Gotham",
             verb: "call",
             action: "Batman",
             with: "Comissioner Gordon",
           },
           {
             taskId: 2,
-            happening: "breaks into Arkham",
             verb: "refuel",
             action: "the Batmobil",
             with: "Alfred",
           },
           {
             taskId: 3,
-            happening: "is starting a gang war",
             verb: "Call",
             action: "Robin",
             with: "Robin",

```

Comment:
would this not suit better in a separate file like a JSON file? Rather than in `App.js`

Reply:
Yes I am doing the frontend first before I build the API and Database